python_modules/dagster/dagster/_core/workspace/permissions.py

```diff
@@ -16,6 +16,7 @@ class Permissions(str, Enum):
     WIPE_ASSETS = "wipe_assets"
     LAUNCH_PARTITION_BACKFILL = "launch_partition_backfill"
     CANCEL_PARTITION_BACKFILL = "cancel_partition_backfill"
+    ADD_DYNAMIC_PARTITION = "add_dynamic_partition"
```

Comment:
suggestion: EDIT_DYNAMIC_PARTITIONS? i bet you will be able to update and remove them too? unless those permissions could be separate

Reply:
This is a good suggestion--changed it to this. I've also added the test as you suggested